src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+
+import duke.DukeException;
+import duke.Ui;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Class containing data and methods specific to a Deadline command
+ */
+public class DeadlineCommand extends Command {
+
+    public DeadlineCommand(String[] command) {
+        super.command = command;
+    }
+
+    /**
+     * Creates Deadline object from commands provided during initialisation of the DeadlineCommand object, adds it to
+     * the data file and prints the added Deadline object.
+     *
+     * @throws DukeException if arguments provided to the DeadlineCommand object are invalid
+     */
```

Comment:
Similar to before, would it be better to use a short summary for the first line before explaining the method in detail?

Reply:
Yes, I think I could work a little on the descriptions on my JavaDocs as well as fixing the punctuation marks issue on the parameter fields, which I didn't realise was a rule until I reviewed someone else's code as well. Thank you for your feedback and I will take all of your recommendations into consideration!